README.md

```diff
@@ -24,10 +24,11 @@ la Liga de Campeones (cinco) y en más temporadas de la Liga de Campeones (cinco
 'Spanish'
 ```
 
-Language detection is not magic. It's just a matter of counting how many of the most common words in a language are included in certain text. If given text contains words of different languages, the detector should return the language that contain most words in given text.
+The most common words in every language are counted in order to find the return value, which is the determined language. If given text contains words of different languages, the detector returns the language that contain most words in given text.
 
-You are responsible of defining the set of the most used words in each of the languages you want to support. As a requirement of this group work, we ask you to support at least 3 languages, including: english, spanish and german.
-
-**Ideas for enhancements in case of extra time:**
-- Solve with regexes (adjusting and/or adding tests might be needed for this)
-- Stats (What word is most common, percentage of languages etc etc)
\ No newline at end of file
+- Split function used in order to make user's input into a working list. 
```

Comment:
Good idea for including this, but maybe it would've been handier to put it as comments inside the code? Although the code itself is easy enough to follow it'd be much more useful if I read what the next block of code does before I review it.